recipes/srm-ifce/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+mkdir build
+cd build
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    # Fix up CMake for using conda's sysroot
+    # See https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html?highlight=cmake#an-aside-on-cmake-and-sysroots
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_TOOLCHAIN_FILE=${RECIPE_DIR}/cross-linux.cmake")
+else
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}")
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}")
+fi
+
+cmake -LAH \
+    "${CMAKE_PLATFORM_FLAGS[@]}" \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DLIB_SUFFIX="" \
+    ..
+
+make -j${CPU_COUNT}
+make install
```

Comment:
The tests provided in the package don’t seem to work despite the final binaries being okay. I’ll look into it at a later date.